sites/public/pages/_app.tsx

```diff
@@ -37,10 +37,9 @@ function BloomApp({ Component, router, pageProps }: AppProps) {
     addTranslation(translations.general, true)
     if (locale && locale !== "en" && translations[locale]) {
       addTranslation(translations[locale])
-    }
-
-    if (overrideTranslations[locale]) {
-      addTranslation(overrideTranslations[locale])
+      if (overrideTranslations[locale]) {
+        addTranslation(overrideTranslations[locale])
+      }
     }
```

Comment:
@pbn4 , we actually want this update from Alameda. Can you make sure this stays.